README.md

```diff
@@ -15,47 +15,43 @@ Add the plugin and repositories listing to `project.clj`. **NB: You need to add
 :plugins [[s3-wagon-private "1.2.0"]]
 ```
 
-You can store credentials either in an encrypted file or as
-environment variables. For the encrypted file, add this to
-`project.clj`:
+To authenticate to the S3 bucket, you can either use any of the AWS SDK credential providers or store credentials in an encrypted file.
 
-```clj
-:repositories [["private" {:url "s3p://mybucket/releases/" :creds :gpg}]]
-```
+- Using one of the AWS SDK [chained provider class][chained-provider-class] credential providers:
 
-And in `~/.lein/credentials.clj.gpg`:
+     Add the following to `project.clj`:
 
-```
-{"s3p://mybucket/releases" {:username "AKIA2489AE28488"
-                            :passphrase "98b0b104ca1211e19a6c"}}
-```
+     ```clj
+     :repositories [["private" {:url "s3p://mybucket/releases/" :no-auth true}]]
+     ```
 
-The username and passphrase here correspond to the AWS Access Key and Secret
-Key, respectively.
+     An excerpt of the most commonly used credential providers:
+     - Environment Variables - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+     - Java System Properties - `aws.accessKeyId` and `aws.secretKey`
+     - Credential profiles file at the default location (~/.aws/credentials) with the [AWS Credentials File Format][credentials-file-format]. To use a particular profile, specify the env var `AWS_PROFILE` or a Java system property of `aws.profile`, otherwise, the fallback will be the default profile name (`"default"`)
```

Comment:
Got it. Yes the version incompatibilities have been a plague. #33 has some of the issues I experienced previously. Getting fully specified to the later version of the SDK doesn't seem too crazy since it doesn't seem like we have everything straightened out in the pom.xml